ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ManagementResources.scala

```diff
@@ -246,7 +235,7 @@ class ManagementResources(processCounter: ProcessCounter,
         val validationResult = processResolving.validateBeforeUiResolving(process)
         val canonical = processResolving.resolveExpressions(process, validationResult.typingInfo)
         val canonicalJson = ProcessMarshaller.toJson(canonical).spaces2
-        (managementActor ? Test(id, canonicalJson, TestData(testData), user, ManagementResources.testResultsVariableEncoder)).mapTo[TestResults[Json]].flatMap { results =>
+        (managementActor ? Test(id, canonicalJson, TestData(testData, 10), user, ManagementResources.testResultsVariableEncoder)).mapTo[TestResults[Json]].flatMap { results =>
```

Comment:
probably maxRows in testData should be get from config

Reply:
yes, thanks...